app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Please update the association between the `Model` and the `User` with that any model belongs_to :author, class_name: `User` so it does make sense 🙏
> Kindly make sure to apply this to all other models